intervals_2.html

```diff
@@ -985,24 +974,16 @@
     }
 
     function saveAsPdf() {
-      const pdfResultsContainer = document.createElement('div');
-      const container = document.getElementById('container');
-      const resultsContainer = document.getElementById('results-container');
-      container.appendChild(pdfResultsContainer);
-      pdfResultsContainer.appendChild(resultsContainer);
-      pdfResultsContainer.classList.add('pdf-results-container');
-
-      const generatePdfButton = document.querySelector('.generate-pdf-button');
-      generatePdfButton.classList.add('remove-button');
-      setTimeout( () => {
-        generatePdfButton.classList.remove('remove-button');
-        pdfResultsContainer.classList.remove('pdf-results-container');
-      }, 250);
-      html2canvas(document.querySelector('.pdf-results-container'), {
+      html2canvas(document.getElementById('results-container'), {
         onrendered: function(canvas) {
           const img = canvas.toDataURL("image/png");
           const doc = new jsPDF();
-          doc.addImage(img, 'PNG', 0, 0, 180, 150);
+          if (window.screen.width <= 600) {
```

Comment:
just a heads up, the fact that this needs to be updated whenever we change the media query is an easy thing to miss. e.g. new collaborators would be very likely to make that mistake. that said, there are no easy solutions in this context AFAIK. found a good article about it here though: https://www.falldowngoboone.com/blog/share-variables-between-javascript-and-css/

also learned about `matchMedia` while looking around, which would be nice to use for this but not strictly necessary: https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia